lighthouse-core/test/report/html/renderer/details-renderer-test.js

```diff
@@ -491,6 +491,37 @@ describe('DetailsRenderer', () => {
       assert.equal(sourceLocationEl.getAttribute('data-source-column'), `${sourceLocation.column}`);
     });
 
+    it('renders source-location values using source map data', () => {
```

Comment:
may need a few more tests to hit the different cases, e.g. when there's no source map and the `title` is appended to the title `renderTextURL()` produces, we'd want to be sure to get an alert if changes to `renderTextURL` broke the rendering of `source-location`.